apps/sms/services/js/message_manager.js

```diff
@@ -91,16 +135,20 @@ var MessageManager = {
     }
   },
 
+  /**
+   * Function to obtain all the Threads and perform a function 
```

Comment:
In addition to that also take a look at [1] to see how we can document complex object params.

[1] http://usejsdoc.org/tags-param.html#parameters-with-properties